server/tests/e2e/maps.e2e-spec.ts

```diff
@@ -518,6 +520,35 @@ describe('Maps', () => {
             }
         });
 
+        run4 = await prisma.run.create({
+            data: {
+                map: { connect: { id: map1.id } },
+                user: { connect: { id: user2.id } },
+                trackNum: 1,
+                zoneNum: 1,
+                ticks: 200000,
+                tickRate: 100,
+                flags: 123,
+                file: '',
+                hash: '35a3a113d817ad904d471ffffb6a966cb241a350',
```

Comment:
Just randomly searched the existing code for this exact hash and it is indeed being used somewhere else. Please don't just duplicate data that's obviously supposed to be random, it's bound to end up breaking unique constraints in the DB at some point.

Reply:
I'm not going to check every test in this PR, please look through and make sure everything is unique yourself

Reply:
How should i generate the hashes?

Reply:
Oh good question, these should be genuine SHA1 hashes. Easiest way is to just use an online tool like [here](https://md5decrypt.net/en/Sha1). Interestingly enough, the hash here decodes to the word "ostriches" :)